symbulate/results.py

```diff
@@ -309,15 +324,72 @@ def plot(self, type=None, alpha=None, normalize=True, jitter=False, **kwargs):
                 raise Exception("Histogram must have type='impulse' or 'bar'.")
         elif dim == 2:
             x, y = zip(*self)
+
+            x_count = count_var(x)
+            y_count = count_var(y)
+            x_height = x_count.values()
+            y_height = y_count.values()
+            
+            discrete_x = sum([(i > 1) for i in x_height]) > .8 * len(x_height)
+            discrete_y = sum([(i > 1) for i in y_height]) > .8 * len(y_height)
+
+            if type is None:
```

Comment:
I'm not sure I agree with these defaults. I think scatterplots should be default for 2 variables. The other plots don't work well if you try to superimpose multiple plots on the same plot (i.e., if you call `.plot()` multiple times for different simulations)

@kevindavisross What do you think?